<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>543</width>
    <height>468</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_10">
    <item>
     <widget class="QLabel" name="label_fpn_width">
      <property name="text">
       <string>Width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_width"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>20</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_12">
    <item>
     <widget class="QLabel" name="label_fpn_height">
      <property name="text">
       <string>Height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_height"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>70</y>
     <width>125</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_13">
    <item>
     <widget class="QLabel" name="label_fpn_bitshift">
      <property name="text">
       <string>BitShift</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_fpn_bitshift"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>70</y>
     <width>143</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_14">
    <item>
     <widget class="QLabel" name="label_fpn_pixeloffset">
      <property name="text">
       <string>PixelOffset</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_fpn_pixeloffset"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>143</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_15">
    <item>
     <widget class="QLabel" name="label_fpn_inputformat">
      <property name="text">
       <string>InputFormat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_fpn_inputformat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>120</y>
     <width>149</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_16">
    <item>
     <widget class="QLabel" name="label_fpn_outputformat">
      <property name="text">
       <string>OutputFormat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_fpn_outputformat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>170</y>
     <width>137</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_17">
    <item>
     <widget class="QLabel" name="label_fpn_dataformat">
      <property name="text">
       <string>DataFormat</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_fpn_dataformat"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>170</y>
     <width>95</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_18">
    <item>
     <widget class="QLabel" name="label_fpn_cfa">
      <property name="text">
       <string>CFA</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_fpn_cfa"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_9">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_19">
    <item>
     <widget class="QLabel" name="label_fpn_start_x">
      <property name="text">
       <string>start_x</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_start_x"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_10">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>220</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_20">
    <item>
     <widget class="QLabel" name="label_fpn_start_y">
      <property name="text">
       <string>start_y</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_start_y"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>270</y>
     <width>105</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_21">
    <item>
     <widget class="QLabel" name="label_fpn_roi_width">
      <property name="text">
       <string>roi_width</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_roi_width"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_12">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>270</y>
     <width>111</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_22">
    <item>
     <widget class="QLabel" name="label_fpn_roi_height">
      <property name="text">
       <string>roi_height</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_roi_height"/>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pushButton_fpn_calculate">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Calculate</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_13">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>320</y>
     <width>181</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_23">
    <item>
     <widget class="QLabel" name="label_fpn_temporal_noise">
      <property name="text">
       <string>TemporalNoise</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_temporal_noise"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_14">
   <property name="geometry">
    <rect>
     <x>154</x>
     <y>320</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_24">
    <item>
     <widget class="QLabel" name="label_fpn_total">
      <property name="text">
       <string>FPN_total</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_total"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_15">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>370</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_25">
    <item>
     <widget class="QLabel" name="label_fpn_v_level">
      <property name="text">
       <string>fpn_v_level</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_v_level"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_16">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>141</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_26">
    <item>
     <widget class="QLabel" name="label_fpn_h_level">
      <property name="text">
       <string>fpn_h_level</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_h_level"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_17">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>370</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_27">
    <item>
     <widget class="QLabel" name="label_fpn_v_max">
      <property name="text">
       <string>fpn_v_max</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_v_max"/>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_18">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>420</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_28">
    <item>
     <widget class="QLabel" name="label_fpn_h_max">
      <property name="text">
       <string>fpn_h_max</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_fpn_h_max"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
